### PR TITLE
Design speed recognition

### PR DESCRIPTION
**Pull request by ThomasintAnker**
Added the speed recognition chapter to the design document, please review.

### CODE REVIEW

**Comment by ThomasintAnker** [comment]
Currently adding an appendix for the calculations required for the speed calculation so that will be added somewhere later today/tomorrow

File: Documentation/DSD_SpeedCamera/content/speed_calculation.tex [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+\section{License plate-based speed measuring}
+Thirdly, the speed of a vehicle can be calculated by tracking the license plate of a vehicle. This due to the fact that all 
+European license plates are created using the same standard measurments. It allows a calculation based on the position of the 
+license plate, and can be used to calculate the distance between the camera and the license plate, which in turn can be used 
+to calculate the speed at which the vehicle is moving.
+
+\section{Conclusion}
+In short, the recommended way of calculating the speed of a vehicle is by using the VASCAR speed measuring method. This because 
+the limited space given in the requirements prevents the use of the camera-only speed measuring method. Furthermore, even
```
Comment (Nike2002) [inline Documentation/DSD_SpeedCamera/content/speed_calculation.tex:45]: Kan je even uitleggen of je het via de fixed point method gaat doen of via de object tracking. Daar zie ik nog geen verschil tussen.

**Review by Nike2002 — CHANGES_REQUESTED** [review]

File: Documentation/DSD_SpeedCamera/content/speed_calculation.tex (hunk 1 above)
Comment (ThomasintAnker) [reply Documentation/DSD_SpeedCamera/content/speed_calculation.tex:45]: Staat in het paragraaf eronder

**Review by maxlintelo — APPROVED** [review]
Approved for deadline

**Pull request merged**